src/main/java/com/textmessenger/controller/PostController.java

```diff
@@ -73,6 +73,7 @@ public ResponseEntity retwitePost(@PathVariable("id") User user, @PathVariable("
 
   @GetMapping("/{id}")
   public ResponseEntity getPostById(@PathVariable("id") long id) {
-    return ResponseEntity.status(200).body(postService.getPostById(id));
+    System.out.println(id);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this use of System.out or System.err by a logger. ([squid:S106](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS106))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=186&issues=AWXYRrclashydt7kRdu-&open=AWXYRrclashydt7kRdu-)

[ISSUE_KEY]: AWXYRrclashydt7kRdu-